packages/field/src/useField.ts

```diff
@@ -42,23 +42,31 @@ export const useField = ({
   );
 
   const getInputProps = useCallback<IUseFieldReturnValue['getInputProps']>(
-    ({ id = inputId, ...other } = {}) => {
-      const describedBy = [];
+    ({ id = inputId, 'aria-describedby': ariaDescribedBy, ...other } = {}) => {
+      const getDescribedBy = () => {
+        if (ariaDescribedBy) {
+          return ariaDescribedBy;
```

Comment:
Is there a scenario where hint or message IDs should be omitted given the custom `aria-describedby`? 👀

Reply:
There's no substantial change here. The consumer was always able to override `aria-describedby` via `...other`. Now we're just reserving the compute for the majority of cases where it is not overriden. In this PR, `useCombobox` "knows better" about the associated hint+message IDs and overrides `aria-describedby` accordingly. The same thing happens on this getter with `aria-labelledby`, but there's no need or opportunity to optimize.